peachjam/models/judgment.py

```diff
@@ -261,6 +261,13 @@ class Judgment(CoreDocument):
     )
     hearing_date = models.DateField(null=True, blank=True)
 
+    auto_assign_title = models.BooleanField(
+        help_text=_("Whether or not the system should assign the title"),
+        null=True,
+        blank=True,
```

Comment:
drop both of these, so it must be either True or False, with a default of True. It'll backfill with True.